### DISCUSSION TITLE
Connecting a provider with another provider using .family

### DISCUSSION THREAD

**Discussion by toshi-kuji** [rrousselGit/riverpod#1085] [Q&A] [answered]
Hi,

I was reading the "family" portion of the document and couldn't fully picture the usage of the description below.

> Some common use-cases for family would be:
> - Connecting a provider with another provider without having access to its variable.

https://riverpod.dev/docs/concepts/modifiers/family/

Is there any good example (whether official or third) describing this use case?

**Comment by rrousselGit** [comment, marked as answer]
I don't think this line is legitimate after all. I'll remove it

**Reply by toshi-kuji** [reply]
Thank you for your prompt answer! Then I will ignore this line for the translation.
